Build/start.sh

```diff
@@ -48,20 +48,21 @@ PREFERRED_RPROVIDER_wpa-supplicant = "sterling-supplicant-lwb"
 PREFERRED_RPROVIDER_wpa-supplicant-cli = "sterling-supplicant-lwb"
 PREFERRED_RPROVIDER_wpa-supplicant-passphrase = "sterling-supplicant-lwb"
 PREFERRED_RPROVIDER_wireless-regdb-static = "wireless-regdb"
+
+MACHINE_FEATURES_append = " docker"
+DISTRO_FEATURES_append = " virtualization"
```

Comment:
I tried this as well when I was looking into it but there was no `docker `  binary in my rootfs  :thinking:  I assume you verified and double checked that it is working, right?

Reply:
The binary is being installed in `mistysom-image.bbappend` which is also part of this pull request. 
I confirm that the binary exists in rootfs. I will test it on a physical G2L soon.

Reply:
please test this (and other changes, too) before merging into `develop`.